examples/default-values-for-function-parameters/default_values_for_function_parameters.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE demonstrates how to provide and use the default value of function parameters in Ballerina.
-keywords: ballerina, ballerina by example, bbe, default value, functions
+description: This BBE demonstrates defining default parameter values in a function using the preceding parameter value as the default parameter value in Ballerina.
```

Comment:
Should we always say "this BBE demonstrates"? This is used for google searches right? @hasithaa

Reply:
No need to use it always, But better if we can have a full sentence.